sys-libs/zlib/files/zlib-1.2.11-configure-fix-AR-RANLIB-NM-detection.patch

```diff
@@ -0,0 +1,74 @@
+From 6221a72528b30e05941309e49808da3ef28da115 Mon Sep 17 00:00:00 2001
```

Comment:
Please include a link to relevant Gentoo bugs and upstream references.

here, I'd include two lines at the top: one for the mailing list submission and one for the Gentoo bug.

Reply:
Done, Thanks!